inst/www/shared/shiny.js

```diff
@@ -3477,7 +3466,6 @@ $(document).on('click', '.slider-animate-button', function(evt) {
           val.to = Math.min(slider.result.max, slider.result.to + slider.options.step);
 
         slider.update(val);
-        forceIonSliderUpdate(slider);
```

Comment:
Oh. I thought you were hoping to get us out of needing to run grunt locally. If we're not aspiring for that, then the main advantage of having Travis do this is, what, that we avoid merge conflicts when two different PRs "simultaneously" change JS code?